tox.ini

```diff
@@ -23,8 +23,9 @@ commands =
     fix: ruff format {posargs:docs}
     fix: ruff check --fix {posargs:docs}
 
-[testenv:nb-tester]
+[testenv:tests]
 deps =
   -e scripts/nb-tester
+  -e scripts/image-tester
   pytest
-commands = pytest scripts/nb-tester
+commands = pytest scripts/nb-tester scripts/image-tester
```

Comment:
Interestingly this seems to work fine but the output shows the tests as if they were all in the `scripts/nb-tester` folder :thinking: 

```
scripts/nb-tester/test/test_get_notebook_jobs.py .........               [ 90%]
scripts/nb-tester/test/test_validate_image.py .                          [100%]
```

I think we can solve it by just running `pytest` instead of `pytest scripts/nb-tester scripts/image-tester` but perhaps there's a better approach.

Reply:
Just running `pytest` is fine by me